lib/utils.pm

```diff
@@ -1156,11 +1154,32 @@ sub reconnect_s390 {
     }
     else {
         my $r = wait_serial($login_ready, 300);
-        if ($r =~ qr/Welcome to SUSE Linux Enterprise 15/) {
-            record_soft_failure('bsc#1040606');
-        }
-        elsif (is_sle) {
-            $r =~ qr/Welcome to SUSE Linux Enterprise Server/ || die "Correct welcome string not found";
+        if (is_sle) {
+            if ($r =~ qr/Welcome to SUSE Linux Enterprise Server/) {
+                return;
+            }
+            else {
+                opensusebasetest::record_info('Welcome message not found', 'Welcome string not found, try ssh to see what is going on...');
```

Comment:
opensusebasetest::record_info? The command record_info comes from the testapi itself. Where have you found this?

Reply:
really good question =)